test/js-api/jsapi.js

```diff
@@ -728,27 +728,27 @@ test(() => {
-    assertInstantiateError([new ArrayBuffer("hi!")], CompileError, "assertInstantiateError([ArrayBuffer('hi!')] => CompileError)");
-    assertInstantiateError([importingModule], TypeError, "assertInstantiateError([importingModule] => TypeError)");
-    assertInstantiateError([importingModule, null], TypeError, "assertInstantiateError([importingModule, null] => TypeError)");
-    assertInstantiateError([importingModuleBinary, null], TypeError, "assertInstantiateError([importingModuleBinary, null] => TypeError)");
-    assertInstantiateError([emptyModule, null], TypeError, "assertInstantiateError([emptyModule] => TypeError)");
-    assertInstantiateError([importingModuleBinary, null], TypeError, "assertInstantiateError([importingModuleBinary, null] => TypeError)");
-    assertInstantiateError([importingModuleBinary, undefined], TypeError, "assertInstantiateError([importingModuleBinary, undefined] => TypeError)");
-    assertInstantiateError([importingModuleBinary, {}], TypeError, "assertInstantiateError([importingModuleBinary, {}] => TypeError)");
-    assertInstantiateError([importingModuleBinary, {"":{g:()=>{}}}], LinkError, "assertInstantiateError([importingModuleBinary, {'':{g:()=>{}}}] => LinkError)");
-    assertInstantiateError([importingModuleBinary, {t:{f:()=>{}}}], TypeError, "assertInstantiateError([importingModuleBinary, {t:{f:()=>{}}}] => TypeError)");
-    assertInstantiateError([complexImportingModuleBinary, null], TypeError, "assertInstantiateError([complexImportingModuleBinary, null] => TypeError)");
-    assertInstantiateError([complexImportingModuleBinary, undefined], TypeError, "assertInstantiateError([complexImportingModuleBinary, undefined] => TypeError)");
-    assertInstantiateError([complexImportingModuleBinary, {}], TypeError, "assertInstantiateError([complexImportingModuleBinary, {}] => TypeError)");
-    assertInstantiateError([complexImportingModuleBinary, {"c": {"d": scratch_memory}}], TypeError, "assertInstantiateError([complexImportingModuleBinary, {'c': {'d': scratch_memory}}] => TypeError)");
+    assertInstantiateError([], TypeError, "instantiate with no args throws TypeError)");
+    assertInstantiateError([undefined], TypeError, "instantiate on undefined throws TypeError)");
+    assertInstantiateError([1], TypeError, "instantiate on a number throws TypeError)");
+    assertInstantiateError([{}], TypeError, "instantiate on empty object throws TypeError)");
+    assertInstantiateError([new Uint8Array()], CompileError, "instantiate on empty Uint8Array throws CompileError)");
+    assertInstantiateError([new ArrayBuffer()], CompileError, "instantiate on empty ArrayBuffer throws CompileError)");
+    assertInstantiateError([new Uint8Array("hi!")], CompileError, "instantiate on Uint8Array('hi!') throws CompileError)");
+    assertInstantiateError([new ArrayBuffer("hi!")], CompileError, "instantiate on ArrayBuffer('hi!') throws CompileError)");
```

Comment:
There's an extra closing parenthesis inside the strings above (but not the ones below). Is that unintentional?